src/components/ga4/EventBuilder/ValidateEvent/handlers/formatCheckLib.ts

```diff
@@ -0,0 +1,116 @@
+import { ValidationMessage } from "../../types"
+
+const RESERVED_EVENT_NAMES = [
+    "ad_activeview", "ad_click", "ad_exposure", "ad_impression", "ad_query",
+    "adunit_exposure", "app_clear_data", "app_install", "app_update",
+    "app_remove", "error", "first_open", "first_visit", "in_app_purchase",
+    "notification_dismiss", "notification_foreground", "notification_open",
+    "notification_receive", "os_update", "screen_view", "session_start",
+    "user_engagement"
+]
+const RESERVED_USER_PROPERTY_NAMES = [
+    "first_open_time", "first_visit_time", "last_deep_link_referrer", "user_id",
+    "first_open_after_install"
+]
+
+export const formatCheckLib = (payload) => {
```

Comment:
Please add comment to public methods. What does `Lib` in `formatCheckLib` name stand for?

Reply:
Added comment!

Lib stands for library, since this method provides a ton of validations. Again I was following naming conventions from the internal tool and am happy to rename if you'd like!